src/test/database/fakeListRemote.spec.ts

```diff
@@ -0,0 +1,103 @@
+import * as pathLib from "path";
+import { expect } from "chai";
+
+import { ListRemote } from "../../database/listRemote";
+
+export class FakeListRemote implements ListRemote {
```

Comment:
Document what this class is for. It took me a while to realize that this is only a fixture for testing, not actually testing "ListRemote".

Reply:
Done.